driver/sql/postgres/eventstore_test.go

```diff
@@ -184,7 +185,7 @@ func TestEventStore_AppendTo(t *testing.T) {
 
 		payloadConverter, messages := mockMessages(ctrl)
 
-		dbMock.ExpectExec(`INSERT(.+)VALUES \(\$1,\$2,\$3,\$4,\$5\),\(\$6,\$7,\$8,\$9,\$10\),\(\$11(.+)`).
+		dbMock.ExpectExec(`INSERT(.+)VALUES \(\$1,\$2,\$3,\$4,\$5\,\$6,\$7,\$8\),\(\$9,\$10,\$11,\$12,\$13,\$14,\$15,\$16\),\(\$17(.+)`).
```

Comment:
Could you please explain this?

Reply:
L186 generates 3 goengine messages, and this is a regex that will match the three inserts that will be executed in L193. 

This is from my understanding of how https://github.com/DATA-DOG/go-sqlmock works. Had to add the parameters since we added more aggregate columns.

Reply:
:+1: Thank you for the explanation:) Makes a bit more sense now.